Articles/ShortArticles/TheDeveloperCertificateOfOrigin.md

```diff
@@ -65,14 +65,14 @@ However, this MAY be implemented as a Contributor License Agreement (CLA), or ot
 
 A few issues must be considered when adopting the DCO for accepting contributions.
 First, since anyone can use any committer name and email address they want with a Git commit (e.g. using `git commit --amend --author "<any-author>"` and therefore impersonate someone else), greater assurance that the person signing off on the DCO is actually the author of the commit can be added by requiring commits to be GPG signed using the `-S` option with `git commit -s -S`<sup>[11]</sup>.
-(However, requiring GPG signing of all commits adds a lot of extra overhead and complexity which may be too much for many developers and therefore more limited usages of GPG signing are possible to provide some elevated assurances <sup>[12]</sup>.
+(However, requiring GPG signing of all commits adds a lot of extra overhead and complexity, which may be too much for many developers and therefore, more limited usages of GPG signing are possible to provide some elevated assurances <sup>[12]</sup>.
```

Comment:
Does a comma at "... for many developers and therefore, more limited usages ..." make sense?  I think, if you add a comma after "therefore" you need to add it before as well like:

"... for many developers and, therefore, more limited usages ..."

See:

* https://english.stackexchange.com/questions/246304/proper-use-of-comma-before-and-after-therefore/246385#:~:text=In%20your%20sample%20sentence%2C%20therefore,I%20need%20to%20respect%20him.

But I think it would be correct to leave the commas out around "therefore".

Other opinions?

Reply:
I agree with you here. I believe I misread this portion and my brain automatically went into "comma" mode, but I dont' think it's necessary in this instance.

Reply:
Thanks! Should be resolved in my last commit 9ec2820627836ebe79c37b588db6556142b06f83.